tests/php/modules/contact-form/test_class.grunion-contact-form.php

```diff
@@ -46,7 +46,7 @@ public function setUp() {
 		$this->post = $post;
 
 		// Initialize plugin
-		$this->plugin = new Grunion_Contact_Form_Plugin;
+		$this->plugin = Grunion_Contact_Form_Plugin::init();
```

Comment:
Why is this part of this PR?

Reply:
The class constructor will invariably call `register_block_type()`, leading to duplicated block registration, which causes a warning from the block registry. Arguably, the class c'tor isn't part of this class' public interface anyway, since it's using the singleton pattern to make sure that no more than one instance exist at any given time, so I'm using the (singleton) `::init()` method instead. (I'm also making the class c'tor `protected` to avoid accidental invocation.)

For more context, see https://github.com/Automattic/jetpack/pull/11212#issuecomment-461770189

Reply:
Cool thanks for the explanation.